include/Server.class.php

```diff
@@ -1,453 +1,453 @@
-                    ':port'           => DBConnection::PARAM_INT,
-                    ':private_port'   => DBConnection::PARAM_INT,
-                    ':max_players'    => DBConnection::PARAM_INT,
-                    ':difficulty'     => DBConnection::PARAM_INT,
-                    ':game_mode'      => DBConnection::PARAM_INT,
-                    ':password'       => DBConnection::PARAM_INT,
-                    ':version'        => DBConnection::PARAM_INT,
-                    ':latitude'       => DBConnection::PARAM_STR,
-                    ':longitude'      => DBConnection::PARAM_STR
-                ]
-            );
-        }
-
-        catch (DBException $e)
-        {
-            throw new ServerException(exception_message_db(_('create a server')));
-        }
-
-        if ($result != 1)
-        {
-            throw new ServerException(_h('Could not create server'));
-        }
-
-        return Server::getServer(DBConnection::get()->lastInsertId());
-    }
-
-    /**
-     * Get a server instance by id
-     *
-     * @param int $id
-     *
-     * @return Server
-     * @throws ServerException
-     */
-    public static function getServer($id)
-    {
-        try
-        {
-            $result = DBConnection::get()->query(
-                "SELECT * FROM `{DB_VERSION}_servers`
-                WHERE `id`= :id",
-                DBConnection::FETCH_ALL,
-                [':id' => $id],
-                [":id" => DBConnection::PARAM_INT]
-            );
-        }
-        catch (DBException $e)
-        {
-            throw new ServerException(exception_message_db(_('retrieve a server')));
-        }
-
-        if (!$result)
-        {
-            throw new ServerException(_h("Server doesn't exist."));
-        }
-        else if (count($result) > 1)
-        {
-            throw new ServerException("Multiple servers match the same id.");
-        }
-
-        return new self($result[0]);
-    }
-
-    /**
-     * Get all servers as xml output
-     *
-     * @throws ServerException
-     * @return string
-     */
-    public static function getServersAsXML()
-    {
-        $servers = [];
-        try
-        {
-            $servers = DBConnection::get()->query(
-                "SELECT * FROM `{DB_VERSION}_servers`",
-                DBConnection::FETCH_ALL
-            );
-        }
-        catch (DBException $e)
-        {
-            throw new ServerException($e);
-        }
-
-        $client_coordinates = Server::getIPCoordinatesFromString(Util::getClientIp());
-
-        // build xml
-        $partial_output = new XMLOutput();
-        $partial_output->startElement('servers');
-        foreach ($servers as $server_result)
-        {
-            $server = new self($server_result);
-            $partial_output->insert($server->asXMLFromClientLocation($client_coordinates[0], $client_coordinates[1]));
-        }
-        $partial_output->endElement();
-
-        return $partial_output->asString();
-    }
-}
+<?php
```

Comment:
Why does github show me that the whole file changed?

Reply:
I changed line endings probably...